test/lit/lit.cfg.py

```diff
@@ -8,6 +9,13 @@
 config.test_source_root = os.path.dirname(__file__)
 config.test_exec_root = os.path.join(config.binaryen_build_root, 'test')
 
+# Import relevant environment variables
+def add_env(var):
+    if var in os.environ:
+        config.environment.update({var: os.environ[var]})
+add_env('BINARYEN_PASS_DEBUG')
+add_env('BINARYEN_CORES')
```

Comment:
Could we simply copy the original env? I think that would be safer as it avoids the risk of us forgetting to update things here. Also, I sometimes run tests with a custom env var locally - say, `BINARYEN_PRINT_FULL` which atm would not work.

Reply:
Sure, that seems safe enough and is how we expected this to work in the first place.